roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -105,6 +105,28 @@ def get_var(self, *keys, **kwargs):
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
         return value
 
+    @staticmethod
+    def get_openshift_version(openshift_image_tag):
```

Comment:
Maybe this should have a more specific name, since it strips out version information? It is returning MAJOR.MINOR and throwing away the rest.